### PR TITLE
Feature: Add pipeline to run regression

### PR DESCRIPTION
**Pull request by IrvinGalindo**

### CODE REVIEW

File: .github/workflows/run-regression.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  run-regression:
+    runs-on: windows-latest
+
+    steps:
+      - name: ✅ Checkout Código
+        uses: actions/checkout@v4
+        with:
+          repository: IrvinGalindo/TeachGitActions
+
+      - name: ✅ Installing java 11
```
Comment (checoman7) [inline .github/workflows/run-regression.yml:19]: soy checoman :3

File: .github/workflows/run-regression.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        uses: actions/setup-java@v4
+          with:
```
Comment (VeroGM87) [inline .github/workflows/run-regression.yml]: There's a syntax error

Comment (IrvinGalindo) [reply .github/workflows/run-regression.yml]: fixed

File: .github/workflows/run-regression.yml (hunk 1 above)
Comment (IrvinGalindo) [reply .github/workflows/run-regression.yml:19]: holiii 😄

**Review by checoman7 — APPROVED** [review]

**Pull request merged**